public/index.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <title>Raspberry Pi Simulator</title>
+    <title>Raspberry Pi Azure IoT Web Simulator</title>
+    <meta name="description" content="This is a Web simulator for Raspberry Pi as client and Azure IoT Hub as service. You can enjoy Azure IoT journey without a real device."
+    <link rel="canonical" href="https://azure-samples.github.io/raspberry-pi-web-simulator/" />
```

Comment:
Since we start to use Azure to build the static page, we can configure the homepage in the [`package.json`](https://github.com/Azure-Samples/raspberry-pi-web-simulator/blob/master/package.json#L66). Then use `%PUBLIC_URL%` in this tag. When webpack package the project, it will replace it.